backend/alembic/versions/ff4852b2a6fd_remove_is_company_from_namespace_and_.py

```
@@ -0,0 +1,77 @@
+            existing_ns = session.execute(
+                select(Namespace.id).where(Namespace.name == 'organization_knowledge')
+            ).scalar_one_or_none()
+
+            if not existing_ns:
+                # Create organization_knowledge namespace
+                op.execute(
+                    """INSERT INTO namespace (name, display_name, owner_user_id, visibility, description, is_active, created_at, updated_at)
+                    VALUES ('organization_knowledge', 'company knowledge base', %s, 'private', '', 1, NOW(), NOW())""",
+                    (admin_user,)
+                )
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)